NEW_MODULE_TECH_EVAL.MD

```diff
@@ -62,10 +71,13 @@ Information gathered includes:
     * Weigh in on any subjective criteria.
     * The Technical Council may ask the Evaluator(s) to make changes to the evaluation.  
     * This review may happen either out of band, during the regularly scheduled Technical Council meeting, or a separate meeting scheduled specifically for this purpose.
-1. The Technical Council decides, using its standard protocols, to accept or reject the module.
+1. The Technical Council decides, using its standard protocols, to accept, provisionally accept, or reject the module, or else to defer a decision pending other action.
     * An evaluation where all criteria are met will typically result in the acceptance of a module by the Technical Council.
-    * An evaluation where not all criteria are met will typically result in the rejection of a module by the Technical Council.
-    * When the Technical council's decision is contrary to the evaluation, the Technical Council is required to provide written justification for their decision.
+    * An evaluation where not all criteria are met will typically result in the rejection of a module by the Technical Council. However:
+        * If some of the failures are due to proposed architectural (or other cross-module) changes, the TC may request that Submitter first propose those changes via the RFC process to get sufficient community input. In that situation the TC may defer its decision pending the resolution of the RFC. (See [Before Development](#Before-Development).)
```

Comment:
This scenario -- where defer a decision due to an architectural change -- is a worst-case scenario.  See the "Before Development" section added to the top of this document suggesting that RFC-level issues are raised much earlier.  And the subgroup also agrees that TC needs to do wider communication encouraging RFCs for these issues.

So this bullet point is a last-ditch solution where none of the above has worked and we still get an architectural change in a TCR.  It is a straw man for discussion.  But the subgroup agreed that using a conditional/provisional acceptance process for architectural changes (as we did for mod-fqm) creates its own difficulties and requires not just process (when is the RFC expected, how do we keep track) and difficult consequences (what if the RFC is rejected; how in practice are all of the affected modules changed).

@mweaver-ebsco any initial thoughts on this?

Reply:
This totally makes sense to me. It's nice to have a documented path for things like FQM, which will hopefully help make things go more smoothly